docs/user-manual/en/address-model.md

```diff
@@ -964,3 +965,13 @@ queues created on the matching address. Defaults to 0. Read more about
 `enable-metrics` determines whether or not metrics will be published to any
 configured metrics plugin for the matching address. Default is `true`. Read more
 about [metrics](metrics.md).
+
+`enable-ingress-timestamp` determines whether or not the broker will add its
+timestamp to messages sent to the matching address. When `true` the exact behavior
+will depend on the specific protocol in use. For AMQP messages the broker will
+add a `long` *message annotation* named `x-opt-ingress-timestamp-millis`. For core
+messages (used by the core and OpenWire protocols) the broker will add a property
+named `_AMQ_INGRESS_TIMESTAMP`. For STOMP messages the broker will add a frame
+header named `ingress-timestamp`. The value will be a 64-bit
+[epoch](https://en.wikipedia.org/wiki/Unix_time) timestamp with millisecond
+precision. Default is `false`.
```

Comment:
I'd suggest "The value will be the number of milliseconds since the epoch" in order to be clearer on the units (vs precision, which is different).

(Unix time is in units of seconds, and though currentTimeMillis is in units of milliseconds, it is not guaranteed as being millisecond precision)

Reply:
Will do.